sections/semantics-forms.include

```diff
@@ -7508,7 +7885,9 @@ You cannot submit this form when the field is incorrect.</samp></pre>
     Sometimes, a user has to select one or more items. This example shows such an interface.
 
     <xmp highlight="html">
-      <label for="multiselect">Select the songs from that you would like on your Act II Mix Tape:</label>
+      <label for="multiselect">
+        Select the songs from that you would like on your Act II Mix Tape:
+      </label>
       <select multiple required id="multiselect" name="act2">
         <option value="s1">It Sucks to Be Me (Reprize)</option>
```

Comment:
Typo. "Repi**z**e" should be "Repri**s**e".  Even in American English https://www.merriam-webster.com/dictionary/reprise